### ISSUE TITLE
AttributeError: module 'pyhmy.staking' has no attribute 'get_all_delegation_information'

### ISSUE THREAD

**Issue by lamk0006** [harmony-one/pyhmy#25]
`main_net = 'https://rpc.s0.t.hmny.io'

delegation_information = staking.get_all_delegation_information(page=-1, endpoint=main_net)
print(delegation_information)`

![image](https://user-images.githubusercontent.com/48664931/145630846-575125fa-385c-4e69-9f7d-d44f2bbca3fd.png)

**Comment by johnashu** [comment]
I copy pasted the code and I had no error.. If you installed via pip, it will be an outdated version..

try cloning the repo and installing using `python setup.py`

**Comment by MaxMustermann2** [comment]
`pip install pyhmy` now returns an updated version 0.1.0 by default. If you have any existing versions, please remove them (`pip uninstall pyhmy`) and make sure to clear your cache (`pip cache purge`).

You can see available versions
```bash
$ pip index versions pyhmy
pyhmy (0.1.0)
Available versions: 0.1.0, 0.0.0
  INSTALLED: 0.1.0
  LATEST:    0.1.0
```
